### DISCUSSION TITLE
newbie - I can't install cloudprober

### DISCUSSION THREAD

**Discussion by pasqualeCatucci** [cloudprober/cloudprober#752] [Q&A]
Hi, I have a Thin Client with Xubuntu 24.04
I installed golang with
apt get golang
and verified go with
go version go1.22.2 linux/amd64
I tried

go get github.com/cloudprober/cloudprober
go: go.mod file not found in current directory or any parent directory.
'go get' is no longer supported outside a module.
To build and install a command, use 'go install' with a version,
like 'go install example.com/cmd@latest'
For more information, see https://golang.org/doc/go-get-install-deprecation
or run 'go help get' or 'go help install'.
and

go install github.com/cloudprober/cloudprober@latest
package github.com/cloudprober/cloudprober is not a main package

could anyone help me?
thanks for all

pasqualeC

**Comment by manugarg** [comment]
Hello @pasqualeCatucci,

We will soon fix the binary path (see #690), but for now you can install it like this:

```
go install github.com/cloudprober/cloudprober/cmd@latest
mv $GOPATH/bin/cmd $GOPATH/bin/cloudprober # fix binary name
```

You can also install using brew -- `brew install cloudprober` or download pre-built binaries from the release page:
https://github.com/cloudprober/cloudprober/releases
